Mathlib/CategoryTheory/Grothendieck.lean

```diff
@@ -306,6 +310,56 @@ def pre (G : D ⥤ C) : Grothendieck (G ⋙ F) ⥤ Grothendieck F where
   map_id X := Grothendieck.ext _ _ (G.map_id _) (by simp)
   map_comp f g := Grothendieck.ext _ _ (G.map_comp _ _) (by simp)
 
+section FunctorFrom
+
+variable {E : Type*} [Category E]
+
+/-- Partially apply a functor `H` on the Grothendieck construction of a functor `F : C ⥤ Cat` to
+an object `c : C` to obtain a functor `F.obj c ⥤ E` on the fiber over `c`. -/
+@[simps]
+def apply (H : Grothendieck F ⥤ E) (c : C) : F.obj c ⥤ E where
```

Comment:
Would not it be more simple to define the "inclusion of fibers" functor `F.obj c ⥤ Grothendieck F`, which would make this "apply" unnecessary, because this would just be the precomposition with `H`. (Already, the definition of the `obj` and `map` fields formally looks like the composition of two functors.)

Reply:
Ah, thanks for the suggestion, I completely mised this.